com/tests/test_views.py

```diff
@@ -319,3 +319,11 @@ def test_ics_updated(self):
                 self.valid_payload,
             )
             mocked.assert_called()
+
+
+@pytest.mark.django_db
+def test_feed(client):
+    """Smoke test that checks that the atom feed is working"""
+    resp = client.get(reverse("com:news_feed"))
```

Comment:
```suggestion
    with assertNumQueries(1):
        resp = client.get(reverse("com:news_feed"))
```
Avec un import de `assertNumQueries` en haut du fichier